docs/ru/http-headers.md

```diff
@@ -0,0 +1,193 @@
+# HTTP заголовки
+
+## Список HTTP заголовков
+
+
+Пакет `yiisoft/http` предоставляет инструменты для парсинга и генерирования HTTP заголовков с учётом их особенностей
+
+## Класс Header
+
+Абсолютно любой заголовок в поле заголовков HTTP-сообщения может быть указан несколько раз, даже если это не разрешается
+в RFC самого заголовка. Поэтому в результате парсинга любого заголовка вы получите коллекцию значений.
+
+Базовым классом для коллекции значений заголовков является `\Yiisoft\Http\Header\Header`. Он подходит для заголовков,
+в которых важно соблюдение порядка значений в заданной последовательности. Для сортируемых списков значений выделена
+отдельная коллекция `\Yiisoft\Http\Header\SortableHeader`, а также специальная коллекция
+`\Yiisoft\Http\Header\AcceptHeader` для семейства заголовков `Accept`. В сортируемых коллекциях значения сразу занимают
+места по порядку согласно параметру `q` или иным правилам, описанным в коллекции.
+
+Рассмотрим пример с заголовком `Forwarded` [RFC7239](https://tools.ietf.org/html/rfc7239).\
+Заголовок представляет собой список из пустых значений с возможными параметрами `by`, `for`, `host` и `proto`:
+
+```
+Forwarded: for=192.0.2.43,for="[2001:db8:cafe::17]",for=unknown
+Forwarded: for=192.0.2.60;proto=http;by=203.0.113.43
+```
+
+Следующий код выводит все значения параметров `for`:
+
+```php
+use \Yiisoft\Http\Header\Value\Forwarded;
+
+/** @var \Psr\Http\Message\ServerRequestInterface $request */
+$header = Forwarded::createHeader()->withValues($request->getHeader('Forwarded'));
```

Comment:
How about shorter syntax?

```suggestion
$header = Forwarded::create($request->getHeader('Forwarded'));
```

Reply:
`Forwarded::createHeader` as nameless will be `Value::createValueCollection`, not `Value::createValue`